src/browser/settings.jsx

```diff
@@ -56,6 +71,14 @@ var SettingsPage = React.createClass({
       var currentWindow = remote.getCurrentWindow();
       currentWindow.setAutoHideMenuBar(config.hideMenuBar);
       currentWindow.setMenuBarVisibility(!config.hideMenuBar);
+
+      var autostart = this.state.autostart;
+      appLauncher.isEnabled().then(function(enabled) {
```

Comment:
Well we could do
```
        if (!autostart) {
           appLauncher.disable();
         } else {
           appLauncher.enable();
         }
```
but that might fail if somebody changes the auto-start outside of our app and already has opend the settings page. Probably a cornercase ;)